data.tf

```diff
@@ -0,0 +1,10 @@
+data "template_file" "user_data" {
```

Comment:
Put all resources and data-sources into single `main.tf` file as we usually do for small modules like this one.